src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/GeneralPreferencesPane.java

```diff
@@ -596,6 +599,15 @@ public RestartRequirement onApply(UserPrefs prefs)
             restartRequirement.setUiReloadRequired(true);
          }
 
+         boolean disableRendererAccessibilityPrefValue = disableRendererAccessibility_.getValue();
+         if (disableRendererAccessibilityPrefValue != initialDisableRendererAccessibility_)
+         {
+            if (Desktop.hasDesktopFrame())
```

Comment:
Would this also need `BrowseCap.isElectron()` since this is Electron only? I think this might be an issue with Desktop Pro, which wouldn't have this implemented.

Reply:
Good catch; fixed!